src/mscorlib/shared/System/Runtime/InteropServices/StringBuffer.cs

```diff
@@ -297,5 +297,8 @@ public void Free()
             _buffer = null;
             _length = 0;
         }
+
+        public static implicit operator ReadOnlySpan<char>(StringBuffer stringBuffer)
```

Comment:
I think it would be better to make the cast explicit via `AsReadOnlySpan` method. StringBuffer is not immutable, and so you have to be careful about for how long you are going to use the Span.